src/components/ProfileBackground/index.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { View, Image } from 'react-native';
+
+import profileCircles from 'assets/images/profileCircles.png';
+import defaultProfileImage from 'assets/images/defaultProfileImage.png';
+import styles from './styles';
+
+const ProfileBackground = () => (
+  <View style={styles.container}>
+    <View style={styles.innerContainer}>
+      <Image source={profileCircles} />
+      <Image source={defaultProfileImage} style={styles.profileImage} />
```

Comment:
This should receive the profile image from the logged user and in case of not having one the defaultProfileImage

```suggestion
      <Image source={profileImage || defaultProfileImage} style={styles.profileImage} />
```

Reply:
Yes! This has been changed. It can be seen in the following profile pr